Lets Do This/Controllers/Campaign/LDTCampaignListViewController.m

```diff
@@ -30,9 +30,6 @@ typedef NS_ENUM(NSInteger, LDTLoadError) {
 	LDTLoadErrorReportback
 };
 
-const CGFloat kHeightCollapsed = 150;
-const CGFloat kHeightExpanded = 382;
```

Comment:
No longer used, as all sizes are looking to be defined within `LDTCampaignListCampaignCell`